packages/gatsby-theme-docs/src/hooks/use-release-notes-config.js

```diff
@@ -14,7 +14,7 @@ const useReleaseNotesConfig = () => {
     const product = products?.[0];
 
     const queryString = buildReleaseNotesQueryString(
-      OTHER_GROUP_SITE_PREFIXES.includes(pathPrefix) ? 'other' : 'product',
```

Comment:
wouldn't be better to just change 

```
const OTHER_GROUP_SITE_PREFIXES = [];
```
and leave the logic in place, in case tomorrow they want a microsite to point to "others" tab?

Or, if you don't agree, go ahead and remove the `OTHER_GROUP_SITE_PREFIXES` const

Reply:
good idea :D .. I'll do that
Thank you

Reply:
@gabriele-ct done in [45885a3](https://github.com/commercetools/commercetools-docs-kit/pull/2092/commits/45885a34f8bd2d80fadd11eb1862275a8fa3915e)